baml_language/ARCHITECTURE.md

```
@@ -0,0 +1,791 @@
+6. [Recursive Types](#recursive-types)
+7. [LLM Function Desugaring](#llm-function-desugaring)
+8. [The Scope System](#the-scope-system)
+9. [The `baml_std` Builtins Pipeline](#the-baml_std-builtins-pipeline)
+10. [Testing Infrastructure](#testing-infrastructure)
+11. [Code References](#code-references)
+
+---
+
+## Pipeline Overview
+
+```
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)